Documentation/Contributing/ci-configuration.md

```diff
@@ -0,0 +1,19 @@
+---
+title: CI Configuration
+---
+
+This page contains information regarding the CI configuration used for the Rook project to test, build and release the project.
+
+## Environment Variables / Secrets
+
+* synk (Security Scan):
+    * `SNYK_TOKEN` - API Token for the [snyk security scanner](https://snyk.io/) (workflow file: `synk.yaml`).
+* Testing:
+    * `IBM_INSTANCE_ID`: Used for KMS (Key Management System) IBM Key Protect access (see [`.github/workflows/encryption-pvc-kms-ibm-kp/action.yml`](https://github.com/rook/rook/blob/master/.github/workflows/encryption-pvc-kms-ibm-kp/action.yml)).
+    * `IBM_SERVICE_API_KEY`: Used for KMS (Key Management System) IBM Key Protect access (see [`.github/workflows/encryption-pvc-kms-ibm-kp/action.yml`](https://github.com/rook/rook/blob/master/.github/workflows/encryption-pvc-kms-ibm-kp/action.yml)).
+* Publishing:
+    * `DOCKER_USERNAME` + `DOCKER_PASSWORD`: Username and password of registry.
+    * `DOCKER_REGISTRY`: Target registry namespace (e.g., `rook`)
+    * `TAG_REGISTRY`: Target registry server `docker.io`
```

Comment:
I don't see `TAG_REGISTRY` in the rook secrets, where is this needed?

Reply:
Ops, yeah we have added that in our fork to be able to push images to other registries than `docker.io`. Removed it.